src/main/java/com/clussmanproductions/trafficcontrol/ModBlocks.java

```diff
@@ -137,6 +95,12 @@
 	public static BlockTrafficLight5Upper traffic_light_5_upper;
 	@ObjectHolder("traffic_light_doghouse")
 	public static BlockTrafficLightDoghouse traffic_light_doghouse;
+	@ObjectHolder("traffic_light_one")
+	public static BlockTrafficLightOne traffic_light_one;
```

Comment:
Any reason you decided to go with the textual "one" instead of the numerical "1"?  I think we should switch it to numerical here and in the above @ObjectHolder just to be consistent.

Reply:
Fixed